.github/workflows/ci-generated.yml

```diff
@@ -26,12 +49,10 @@ jobs:
     - name: Commit files
       run: |
         git add .
-        git config --local user.email "action@github.com"
-        git config --local user.name "GitHub Action"
-        git commit -m "Add Generated Files" -a
+        MASTER_REV=$(git log -1 master --pretty=%H)
+        git commit -m "Add Generated Files for ${MASTER_REV}" -a
```

Comment:
nit: Should this amend the merge commit?

Reply:
I'd like to keep them separate, just for clarity